Signal/translations/en.lproj/Localizable.strings

```diff
@@ -499,6 +490,18 @@
 /* No comment provided by engineer. */
 "MESSAGE_COMPOSEVIEW_TITLE" = "New Message";
 
+/* message footer for delivered messages */
```

Comment:
Reminder to push translations right after merging this to give our translators time to retranslate these (make sure any outstanding hotfix branches are merged first).